### PR TITLE
Bakaffa

### PR DESCRIPTION
**Pull request by sologebre**
Please Dorothea could you make review

### CODE REVIEW

File: 4001-5000/LIT4633Chronicle.xml [hunk 1]
```diff
@@ -341,7 +341,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 </div>
                 <div type="textpart" subtype="paragraph" n="14">
                     <ab>ወበዝንቱ፡ ኵሉ፡ ሁከት፡ ኢያንቀልቀለ፡ ልቡ፡ እምትውክልተ፡ እግዚአብሔር።
-                        ወአኀዘ፡ ይጼሊ፡  <persName ref="PRS2377Bakaffa">
```
Comment (thea-m) [inline 4001-5000/LIT4633Chronicle.xml]: Seems better to me with the space

File: 4001-5000/LIT4633Chronicle.xml [hunk 2]
```diff
@@ -880,8 +894,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 </div>
                 
                 <div type="textpart" subtype="paragraph" n="47">
-                    <ab>1ክፍል፡ ፲፪።2ሠረቀ፡ ሐምሌ፡ በዕለተ፡ ሠሉስ።3ወአመ፡ ፪፡ ለሐምሌ፡ ሞተ፡ ባሻ፡ ወልደ፡ ጊዮርጊስ፡ 
-                        ቈለኛ፡ ወተቀብረ፡ በደብረ፡ (G7v) ብርሃን።4ወበውእቱ፡ ወርኅ፡ ተስዕረ1፡ ባሻ፡ አርከሌድስ፡ ወተሠይመ፡ ባሻ፡ አደራ።
+                    <ab>ክፍል፡ ፲፪።2ሠረቀ፡ ሐምሌ፡ በዕለተ፡ ሠሉስ። ወአመ፡ ፪፡ ለሐምሌ፡ ሞተ፡ ባሻ፡ ወልደ፡ ጊዮርጊስ፡ 
+                        ቈለኛ፡ ወተቀብረ፡ በ<placeName ref="LOC2248DabraB">ደብረ፡ ብርሃን።4</placeName>ወበውእቱ፡ ወርኅ፡ ተስዕረ፡
```
Comment (thea-m) [inline 4001-5000/LIT4633Chronicle.xml]: Does the 4 belong here?

File: 4001-5000/LIT4633Chronicle.xml [hunk 3]
```diff
@@ -914,27 +929,34 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                 
                 <div type="textpart" subtype="paragraph" n="50">
-                    <ab> 1ክፍል፡ ፲፭።2ሠረቀ፡ ጥቅምት፡ በዕለተ፡ እሑድ።3ወአመ፡ ፲ወ፪፡ ለጥቅምት፡ አዕረፈ፡ ቄስ፡ ሐፄ፡ ቀውስጦስ፡ 
-                        ዐቢይ1፡ ካህን።4ወእምድኅረ፡ ኀለፈ2፡ ጊዜ፡ ፱፡ ሰዓት፡ ጸውዖ፡ ንጉሥ፡ ለአባ፡ ዘወልደ፡ ማርያም፡ አበ፡ መጉና።5ወነበረ፡ 
-                        ዲበ፡ መንበረ፡ መንግሥት፡ በማዕከላይ፡ ግምብ።6ወነገሮ፡ ከመ፡ የኀሥሥ3፡ ጽድቀ፡ በሕጉ።7ወአስተጋብኦሙ፡ ለካህናት፡ 
+                    <ab>ክፍል፡ ፲፭።2ሠረቀ፡ ጥቅምት፡ በዕለተ፡ እሑድ።3ወአመ፡ ፲ወ፪፡ ለጥቅምት፡ አዕረፈ፡  
+                        <persName ref="PRS12795Qawestos"><roleName type="title">ቄስ፡ ሐፄ፡</roleName> ቀውስጦስ፡ </persName>
+                        ዐቢይ፡ ካህን።ወእምድኅረ፡ ኀለፈ፡ ጊዜ፡ ፱፡ ሰዓት፡ ጸውዖ፡ <persName ref="PRS2377Bakaffa"><roleName type="title">ንጉሥ፡</roleName> </persName>
+                        ለአባ፡ ዘወልደ፡ ማርያም፡ አበ፡ መጉና። ወነበረ፡ 
+                        ዲበ፡ መንበረ፡ መንግሥት፡ በማዕከላይ፡ ግምብ። ወነገሮ፡ ከመ፡ የኀሥሥ3፡ ጽድቀ፡ በሕጉ።7ወአስተጋብኦሙ፡ ለካህናት፡
```
Comment (thea-m) [inline 4001-5000/LIT4633Chronicle.xml]: I don't understand the use of Arabic numerals here and throughout the record - are they sentence numbers? Why does 7 follow 3? If they are sentence numbers, why are they encoded as part of the edited text, without even spaces between them and the words around them?

File: 4001-5000/LIT4633Chronicle.xml [hunk 4]
```diff
@@ -1020,76 +1042,91 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                        <persName ref="PRS2377Bakaffa"><roleName type="title">ንጉሥ፡
+                        </roleName>በካፋ፡ </persName> 
+                        ዐቢይ፡ ኀይል፡ እምዐቢይ፡ እግዚአብሔር፡ ዘዐቢይ፡ ኀይሉ።ምክንያቱሰ፡ አንሥአ፡
+                        ከተማ፡ ነገረ፡ ዘውእቱ፡ ወሬ፡ እንዘ፡ ይብል፡ የሐውር፡   <persName ref="PRS2377Bakaffa"><roleName type="title">ንጉሥ፡
+                        </roleName>  </persName> ውስተ፡ <placeName ref="LOC6186Walqay">ወልቃይት።</placeName> ወበዝ፡ ተስፋ፡ 
+                        ቦአ፡ ወሬኛ፡ ውስተ፡ መዲና፡  ወተኀብአ፡ በቤተ፡  <persName ref="PRS12807ZaraSeyon"><roleName type="title">አዛዢ፡</roleName> ዘርአ፡ ጽዮን።</persName> 
+                        ወአኀዘ፡ ይስሕብ፡ ሰብአ፡ እንዘ፡ ይብል፡ 
+                        ናሁ፡ እነግሥ፡ አነ፡ እስመ፡ በጽሐ፡ ጊዜ፡ መንግሥትየ።ወእሁበክሙ፡ ሢመተ፡ 
+                        ወሢመተ፡ እገሌ፡ እሁብ፡ ለእገሌ።ወበዝ፡ ኀብሩ፡ ሎቱ፡ ብዙኃን፡ ሰብእ፡ ፈድፋደሰ፡ ወዓልቱ፡ ለ<persName ref="PRS12775Mammuye">ማሙዬ።
+                        </persName>
```
Comment (thea-m) [inline 4001-5000/LIT4633Chronicle.xml]: There are no spaces after ።, not only here but again throughout the record

File: 4001-5000/LIT4633Chronicle.xml (hunk 2 above)
Comment (sologebre) [reply 4001-5000/LIT4633Chronicle.xml]: I will delete all Arabic numerals that are not part of the text, but they are footnotes number for the edition.

File: 4001-5000/LIT4633Chronicle.xml (hunk 3 above)
Comment (sologebre) [reply 4001-5000/LIT4633Chronicle.xml]: Here I have only encoded person, places, and other entities. I have not considered the Arabic numbers that refer to the footnotes, so I will delete all Arabic numerals

File: 4001-5000/LIT4633Chronicle.xml (hunk 4 above)
Comment (sologebre) [reply 4001-5000/LIT4633Chronicle.xml]: I have corrected now.

**Pull request merged**